examples/tensorflow/test_tensorflow_examples.py

```diff
@@ -294,3 +296,27 @@ def test_run_translation(self):
             run_translation.main()
             result = get_results(tmp_dir)
             self.assertGreaterEqual(result["bleu"], 30)
+
+    @slow
```

Comment:
So, this was paused for quite a while because the script was taking ages on the `prepare_tf_dataset` step. Thanks to the sleuthing from @Rocketknight1 and the subsequent to_tf_dataset updates and using `with_format("np")` things move a lot faster. Switching to a `hf-internal-testing` dataset and small model meant it runs < 30s.